### PR TITLE
removed data volume legacy app docs

### PR DESCRIPTION
**Pull request by JKashyap96**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

https://jira.kumoroku.com/jira/browse/SUMO-218042

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by JV0812 — APPROVED** [review]

**Comment by JKashyap96** [comment]
Why did the test fail?

error trace:
Error:  Unable to build website for locale en.
Error:  Error: You are trying to create client-side redirections to invalid paths.

**Comment by JKashyap96** [comment]
Once this [JIRA](https://jira.kumoroku.com/jira/browse/SUMO-218042) for app release gets implemented, the docs can be deleted.

**Review by kimsauce — CHANGES_REQUESTED** [review]
@JKashyap96 @JV0812 because you're deleting a doc, you also need to do the following (prevents broken URLs and possible confusion for customers, upholds our SEO): 

- Set up a 301 in cid-redirects.json from the legacy URL to new URL
- Find all instances of the legacy URL in the and replace them with the new URL
- In the newer Data Volume doc, add a note saying the legacy app has been deprecated

Let me know if I can help with these!

**Comment by kimsauce** [comment]
> Thanks @kimsauce! Could you please help @JV0812 make the changes? Also, the app has been deleted from the production environments, so we can merge this.

@JKashyap96 thank you for the update! @JV0812 , I've created guidance [here](https://help.sumologic.com/docs/contributing/remove-doc/) on how to do it.

File: cid-redirects.json [hunk 1]
```diff
@@ -1457,7 +1455,7 @@
   "/cid/10167": "/docs/send-data/hosted-collectors/google-source/gcp-metrics-source",
   "/cid/10168": "/docs/integrations/saas-cloud/pagerduty-v3",
   "/cid/10169": "/docs/integrations/cloud-security-monitoring-analytics/palo-alto-firewall-10",
-  "/cid/1017": "/docs/integrations/sumo-apps/data-volume-legacy",
+  "/cid/1017": "/docs/integrations/sumo-apps/data-volume-legacy : /docs/integrations/sumo-apps/data-volume",
```
Comment (kimsauce) [inline cid-redirects.json:1458]:
```suggestion
  "/cid/1017": "/docs/integrations/sumo-apps/data-volume-legacy" : "/docs/integrations/sumo-apps/data-volume",
```

File: cid-redirects.json [hunk 2]
```diff
@@ -2011,7 +2009,7 @@
   "/cid/5191": "/docs/integrations/containers-orchestration/vmware-legacy",
   "/cid/5192": "/docs/integrations/amazon-aws/cloudfront",
   "/cid/5193": "/docs/integrations/security-threat-detection/cisco-asa",
-  "/cid/5194": "/docs/integrations/sumo-apps/data-volume-legacy",
+  "/cid/5194": "/docs/integrations/sumo-apps/data-volume-legacy :/docs/integrations/sumo-apps/data-volume",
```
Comment (kimsauce) [inline cid-redirects.json:2012]: Need to put each URL in quotes (this should've broken your local build)

```suggestion
  "/cid/5194": "/docs/integrations/sumo-apps/data-volume-legacy" : "/docs/integrations/sumo-apps/data-volume",
```

Comment (kimsauce) [inline cid-redirects.json:2012]: You need to replace the URL, not add to it

```suggestion
  "/cid/5194": "/docs/integrations/sumo-apps/data-volume",
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**